src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -177,9 +176,9 @@ export class DictionaryModal extends React.Component {
           </Modal.Header>
           {errors && <InlineError text={this.errors} />}
           <Modal.Body>
-            <form style={{ marginLeft: '50px', marginRight: '-37%' }}>
-              <div className="row">
-                <div className="col-md-8">
+            <form>
+              <div>
+                <div>
                   <FormGroup style={{ marginTop: '12px' }}>
```

Comment:
Is it possible to transfer this style to the CSS file?

Reply:
Yes, it is but it outside the scope of this ticket.